nece/models.py

```diff
@@ -13,7 +16,30 @@
     from nece.fields.pgjson import JSONField
 
 
-class TranslationModel(models.Model, TranslationMixin):
+class TranslateOptions(object):
+    translatable_fields = ()
+
+    def __init__(self, opts):
+        if opts:
+            items = filter(
+                    lambda x: not x[0].startswith('__'), opts.__dict__.items())
```

Comment:
I don't like this code. Is this hack or not ? optional solution:
There may be a supported field name list and we can save only these fields.